### PR TITLE
[#140342371] Fix unit tests

### PR DESCRIPTION
**Pull request by guillaumedebavelaere**

### CODE REVIEW

File: TradeItIosTicketSDK2/TradeItSDK.swift [hunk 1]
```diff
@@ -9,6 +9,8 @@
     public static let yahooLauncher = TradeItYahooLauncher()
     internal static let linkedBrokerCache = TradeItLinkedBrokerCache()
 
+
+    internal static var _linkedBrokerManager: TradeItLinkedBrokerManager?
```
Comment (jsom) [inline TradeItIosTicketSDK2/TradeItSDK.swift]: How come this got moved up?

Comment (guillaumedebavelaere) [inline TradeItIosTicketSDK2/TradeItSDK.swift]: I had a conflict and used the github resolve tools, I may missed something.

Comment (jsom) [inline TradeItIosTicketSDK2/TradeItSDK.swift]: Oh ok - can you move it down to above `public static var linkedBrokerManager: TradeItLinkedBrokerManager`

Comment (guillaumedebavelaere) [inline TradeItIosTicketSDK2/TradeItSDK.swift]: should be good now

**Pull request merged**